### ISSUE TITLE
GPU Pricing is broken

### ISSUE THREAD

**Issue by mcybz** [doitintl/gcpinstances.info#8]
<img width="1015" alt="image" src="https://user-images.githubusercontent.com/83850054/130993063-fe4472d3-3ab5-4fc8-9930-09827f939ebf.png">

<img width="1426" alt="image" src="https://user-images.githubusercontent.com/83850054/130993144-bc4772bd-1bab-4c1f-85ee-420ca0ba271c.png">

https://gcpinstances.info/?cost_duration=monthly does not matche https://cloud.google.com/compute/gpus-pricing. And yes of course the google page bills in 1GPU increments.

**Comment by mcybz** [comment]
Yeah I hear ya. Its tricky cos its a bundle deal of different SKU's behind the scenes. That said, if you get an a2-megagpu-16 it can only be instantiated with 16 GPUs so its reasonable to treat it as one. This is what the Instance Wizard does and the Cost Calculator.

![image](https://user-images.githubusercontent.com/83850054/131415384-aafcf960-36d5-4e2f-8be1-d3bdbbf74227.png)

**Comment by hadim** [comment]
I would be interested in having the GPU prices too. I would say having another tab listing only the GPU prices without the machine is fine.
